kroxylicious-docs/docs/sasl-inspection-guide/index.adoc

```diff
@@ -11,7 +11,7 @@ include::_modules/sasl-inspection/con-about-{guide}-guide.adoc[leveloffset=+1]
 
 [role="_abstract"]
 This filter inspects the SASL exchange between Kafka Client and Broker and extracts the *authenticated principal*.
```

Comment:
I wonder if it's right to talk about a singleton `authenticated principal` when the relationship is many principals per Subject. We don't want to put it in users minds that there's a single principal per client, even though the Filter currently creates a single principal.

Reply:
Good point, I'll reword.

Aside: In the Oauth bearer case, the JWT may contain group information.  So at some point, it would be good to have the SASL Inspector / SASL Inspector extract Groups too (not for now).

Reply:
How does that look?